crates-reth/testing/eth-tests/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+#![cfg(test)]
+mod tests {
+    use ethers::{core::k256::ecdsa::SigningKey, prelude::*, signers::coins_bip39::English};
+
+    /// Dev node rpc url
+    ///
+    /// Must be running a local node before running tests
+    const RPC_URL: &str = "http://192.168.1.44:8545";
```

Comment:
Change this constant to environment variable with default value http://127.0.0.1:8545